panther_hardware_interfaces/include/panther_hardware_interfaces/motors_controller.hpp

```diff
@@ -58,26 +60,18 @@ class MotorsController
   void Activate();
 
   /**
-   * @brief Updates current Roboteq feedback state (position, velocity, current, flags).
+   * @brief Updates current motors' states (position, velocity, current).
    *
-   * @exception std::runtime_error if current data is timed out or any error flag on Roboteq
-   * driver was set or CAN error was detected
+   * @exception std::runtime_error if CAN error was detected
    */
-  void UpdateSystemFeedback();
+  void UpdateMotorsStates();
 
   /**
-   * @brief Updates one of the current Roboteq driver feedback states (temperature, voltage,
-   * battery current). It has to be called 8 times to update all values. It was separated
-   * to allow higher frequencies of the controller - reading all the values at once takes
-   * some time. By reading values one by one, the required time won't be as long. These values
-   * don't have to be updated that frequently, so having a frequency of controller_frequency/8
-   * shouldn't be a problem.
+   * @brief Updates current Roboteq driver state (flags, temperatures, voltage, battery current)
    *
-   * @exception std::runtime_error if there was an error
-   * @return whether all updates were finished - only one is read every iteration.
-   * Once it is ready, driver state values can be accessed
+   * @exception std::runtime_error if CAN error was detected
    */
-  bool UpdateDriversState();
+  void UpdateDriversState();
 
   const RoboteqData & GetFrontData() { return front_data_; }
   const RoboteqData & GetRearData() { return rear_data_; }
```

Comment:
```suggestion
  const RoboteqData & GetFrontDataRef() { return front_data_; }
  const RoboteqData & GetRearDataRef() { return rear_data_; }
```
I believe we discussed this ref/copy issue previously, but I just suggest adding `Ref` to the method names as a solution. What are your thoughts?

Reply:
I would prefer the regular names (without `Ref`), how about you @KmakD?

Reply:
Don't have a strong opinion about this. I guess with `Ref` we leave less room for confusion later

Reply:
To be honest, I would even prefer to remove reference and just copy, if that can be confusing, instead of adding `Ref` to method names xd I think that it will look strange, and I haven't seen such convention anywhere.